knowledge-base/adr/0031-resilience-options.md

```diff
@@ -0,0 +1,257 @@
+# Resilience Options in the SAP Cloud SDK for JavaScript
+
+## Status
+
+proposed
+
+## Context
+
+### Overview on Request Types
+
+If a user uses the typed clients or the `executeHttpRequest()` method, the SDK creates a different requests.
```

Comment:
There is a mistake in this sentence, but I also don't get the purpose of this sentence? Are the requests different?! Or do you mean multiple?